src/core/worker.js

```diff
@@ -555,12 +452,7 @@ var WorkerMessageHandler = {
             !fullRequest.isRangeSupported) {
           // If stream or range are disabled, it's our only way to report
           // loading progress.
-          fullRequest.onProgress = function (evt) {
-            handler.send('DocProgress', {
-              loaded: evt.loaded,
-              total: evt.total,
-            });
-          };
+          handler.send('DocProgress');
```

Comment:
I am trying to fix a round trip of progress data(from main to worker, back to main) here. As we don't have any progress data at worker side, this message doesn't contain any data, it is just to notify `DocProgress` event.